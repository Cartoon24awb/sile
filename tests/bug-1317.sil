\begin[direction=RTL,papersize=a7]{document}
\font[family=Amiri,size=30pt]
\nofolios
\neverindent
\begin{script}

SILE.documentState.documentClass:declareFrame("other", {
    left = "left(content) + 20%pw",
    right = "right(content) - 20%pw",
    top = "top(content) + 20%ph",
    bottom = "bottom(content) - 20%ph",
    direction = "RTL"
  })

SILE.typesetNaturally(SILE.getFrame("other"), function()
  SILE.typesetter:typeset("عَرَبي")
end)

SILE.typesetter:typeset("عَرَبي")

\end{script}
\end{document}
