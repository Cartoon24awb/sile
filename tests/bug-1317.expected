Set paper size 	209.7637818	297.6377985
Begin page
Mx 	104.1628
My 	105.4894
Set font 	Amiri;30;400;;normal;;RTL
Mx 	14.2800
Mx 	7.7400
Mx 	16.8300
Mx 	14.3100
T	5147 a=14.2800 5064 a=7.7400 430 x=2.1300 y=-13.9200 3119 a=14.1900 x=2.6400 430 x=-2.2800 2055 a=14.3100	(عَرَبي)
Mx 	146.1156
My 	45.9619
Mx 	14.2800
Mx 	7.7400
Mx 	16.8300
Mx 	14.3100
T	5147 a=14.2800 5064 a=7.7400 430 x=2.1300 y=-13.9200 3119 a=14.1900 x=2.6400 430 x=-2.2800 2055 a=14.3100	(عَرَبي)
End page
Finish
